ui/components/MesheryPerformance/PerformanceProfiles.js

```diff
@@ -33,15 +40,20 @@ const styles = (theme) => ({
     flexGrow : 1
   },
   dialogHeader : {
-    backgroundColor : theme.palette.type === 'dark' ? theme.palette.secondary.headerColor : theme.palette.secondary.mainBackground,
+    backgroundColor : theme.palette.type === 'dark' ? theme.palette.secondary.headerColor: theme.palette.secondary.mainBackground,
```

Comment:
All of the style in this data table should be centralized, @sudhanshutech. This is upsetting to see given all of the conversations that we've had.